nav2_velocity_smoother/src/velocity_smoother.cpp

```diff
@@ -119,18 +119,23 @@ VelocitySmoother::on_configure(const rclcpp_lifecycle::State & state)
   node->get_parameter("deadband_velocity", deadband_velocities_);
   node->get_parameter("velocity_timeout", velocity_timeout_dbl);
   velocity_timeout_ = rclcpp::Duration::from_seconds(velocity_timeout_dbl);
+  size_t size = max_velocities_.size();
```

Comment:
I think the `for (unsigned int i = 0; i != 3; i++) {` above needs to be updated based on the size for the full 6DOF